figo/figo.py

```diff
@@ -881,10 +881,14 @@ def get_transactions(self, account_id=None, since=None, count=1000, offset=0,
         if since is not None:
             params['since'] = since
 
-        return self._query_api_object(Transaction, (
-            "/rest/transactions?" if account_id is None else (
-                "/rest/accounts/%s/transactions?" % account_id)) + urllib.urlencode(params),
-                                      collection_name="transactions")
+        params = urllib.urlencode(params)
+
+        if account_id:
```

Comment:
i would check `account_id is not None` ... otherwise you could get false negatives here with `account_id == 0`

same below